### PR TITLE
[20] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by tomike19**
## Task: `Sign-Up Form Validation`
- `Changes you made to given task`

### CODE REVIEW

**Comment by adaugochi** [comment]
Try fixing the issue the automatic code review is pointing at

**Comment by tomike19** [comment]
@adaugochi Please review my changes

File: submissions/kolajoTomike/DOM/newletter.js [hunk 1]
```diff
@@ -17,4 +17,8 @@ let emailRegex = /^\w+[\w-\.]*\@\w+((-\w+)|(\w*))\.[a-z]{2,3}$/
         }
        
     }
-    submitForm.addEventListener('submit', validateForm);
\ No newline at end of file
+    submitForm.addEventListener('submit', validateForm);
+
```
Comment (adaugochi) [inline submissions/kolajoTomike/DOM/newletter.js:21]: Remove new line spaces

File: submissions/kolajoTomike/DOM/signup.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+    let form = document.getElementById("signup_form"),
+    firstName = document.getElementById("firstName"),
+    lastName = document.getElementById("lastName"),
+    phoneNumber = document.getElementById("phoneNumber"),
+    formEmail = document.getElementById("formEmail"),
+    password = document.getElementById("password"),
+    confirmPassword = document.getElementById("confirmPassword");
+  
+  function validation(field, regExp, errorMess1, errorMess2, event) {
+    event.preventDefault();
```
Comment (adaugochi) [inline submissions/kolajoTomike/DOM/signup.js:10]: Remove this

Comment (tomike19) [reply submissions/kolajoTomike/DOM/signup.js]: I have....kindly review again

**Review by adaugochi — APPROVED** [review]

**Pull request merged**